### PR TITLE
Move rafHandle declaration

### PR DESCRIPTION
**Pull request by TheIceCreamBear**

### CODE REVIEW

**Review by ctcuff — APPROVED** [review]

**Review by Armintron — APPROVED** [review]
LGTM, but why lol.

**Comment by TheIceCreamBear** [comment]
More organized on the brain, and it also removes the double `// // comment`

File: src/index.js [hunk 1]
```diff
@@ -28,10 +28,13 @@ const main = async () => {
     keyDown[e.code] = false;
   });
 
-  // // init gl stuff here, like back face culling and the depth test
+  // init gl stuff here, like back face culling and the depth test
   gl.enable(gl.DEPTH_TEST);
   gl.clearColor(0.5, 0.2, 0.7, 1.0);
 
+  // the handle to the current requested animation frame, set later
+  let rafHandle = 0;
```
Comment (Armintron) [inline src/index.js:36]: why cant you just leave it as `undefined`?

**Comment by TheIceCreamBear** [comment]
My justification for it is that it follows a more "normal" language syntax and that I usually like see vars declared before they are used, even if they are used in scoped functions

**Comment by Armintron** [comment]
> My justification for it is that it follows a more "normal" language syntax and that I usually like see vars declared before they are used, even if they are used in scoped functions

You should reply to my comment rather than leaving a comment on the PR.
However I think init to 0 is a little confusing bc it makes you wonder what the var represents; 0 isnt a very "normal" default value for a function expression.
I do not care at all though lmao, just my 2 cents.

**Comment by TheIceCreamBear** [comment]
> > My justification for it is that it follows a more "normal" language syntax and that I usually like see vars declared before they are used, even if they are used in scoped functions
> 
> You should reply to my comment rather than leaving a comment on the PR. However I think init to 0 is a little confusing bc it makes you wonder what the var represents; 0 isnt a very "normal" default value for a function expression. I do not care at all though lmao, just my 2 cents.

NGL, didn't see that you left a comment as a change request, and I also see your justification for it so ill fix that really quick

**Pull request merged**